templates/zerver/help/mention-a-team-member.md

```diff
@@ -0,0 +1,11 @@
+#Mention a Team Member
+
+![Mention a Team Member](/static/images/help/mention_user.png)
+
+When having a conversation in Zulip, it is sometimes necessary to call the attention of another member. You can alert ("``@mention``") other users with a particular stream-message from within the body of the message itself. Type @ and their email address or one of their names, and the system will offer you tab-completions. The completed ``@mention`` will look something like: ``@**name**``, and it will appear as ``@name`` in your posted message.
```

Comment:
"Begin typing" would work great!

In your file you are using \`\` around some terms, but in the original docs they were not used - you can look at the [commit files](https://github.com/zulip/zulip/pull/2459/files) and there will be no Markdown rendering, so you can see what I meant.